lib/linguist/languages.yml

```diff
@@ -41,33 +41,33 @@
   type: programming
   color: "#814CCC"
   extensions:
-  - ".bsl"
-  - ".os"
+    - ".bsl"
+    - ".os"
```

Comment:
Any idea what happened here?

Reply:
Hey @pchaigno, just asked the guys at prisma to go ahead and add the missing license to the grammar repo. Should be up soon!

I believe that's my [prettier](https://github.com/prettier/prettier) settings on vscode that are auto editing those files. I can go ahead and clean them up if you'd like and disable it from now on!